app/views/internal/_provider_event.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= link_to internal_event_path(id: provider_event.readable_id), class: "events-featured__list__item" do %>
```

Comment:
Should this partial just be `_event.html.erb`? I don't think we use the term provider event elsewhere?